PWGHF/D2H/Tasks/taskOmegacSt.cxx

```diff
@@ -29,20 +33,116 @@
 #include "Framework/O2DatabasePDGPlugin.h"
 #include "Framework/runDataProcessing.h"
 #include "ReconstructionDataFormats/DCA.h"
-
-#include "Common/Core/RecoDecay.h"
-#include "Common/Core/trackUtilities.h"
-#include "Common/DataModel/TrackSelectionTables.h"
-
 #include "PWGLF/DataModel/LFStrangenessTables.h"
-
 #include "PWGHF/Core/PDG.h"
 #include "PWGHF/Core/SelectorCuts.h"
 
 using namespace o2;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
 
+namespace o2::aod
+{
+namespace stomegac
```

Comment:
Why is the data model defined directly in the task file?

Reply:
This is the definition of the tree output of that specific task and I followed the example of other tasks having this in the same source file. Are the definitions for tree outputs normally added under `DataModel`?

Reply:
Ok, so we should move the workflow to `TableProducer`. Do we also have to split the table definition into a separate file under `DataModel`?

Reply:
If the tables are (supposed to be) consumed by another workflow, it has to be defined in `DataModel`, if not then it is a tree creator workflow and the tables can stay inside the file.

Reply:
Ok, it falls in the category of tree creator. What should the workflow be called then? data-creator-st-omegac?

Reply:
`tree-creator-omegac-st` and the file should go to `PWGHF/TableProducer`.
(`data-creator` workflows have been introduced with the reduced data format for B mesons, which are a different kind of table producer for self-contained tables consumed by analysis tasks.)